app/controllers/api/v1/search_controller.rb

```diff
@@ -32,7 +32,9 @@ class SearchController < ApiBaseController
           parameter do
             key :name, :facets
             key :in, :query
-            key :description, 'User-supplied list facets and filters, formatted as: "facet_id:filter_value+facet_id_2:filter_value_2,filter_value_3"'
+            key :description, 'User-supplied list facets and filters.  Delimit facets from filters with ":", filter ' \
+                              'values with ";" and facet entries with "|"'
+            key :example, 'facet_id:filter_value|facet_id_2:filter_value_2;filter_value_3'
```

Comment:
I agree - and I did spend some time thinking about this.  The main reason why I didn't change it as it would require us to version the _entire_ API up to `v2`.  For something this small, it seemed like a waste of effort.  Instead, I'm going to open a ticket for a feature request to allow versioning of individual API endpoints.  I'm not sure exactly how we'd do that given the current structure, but it seems like the "whole API" version isn't going to scale well for us.

Reply:
big fan of how you updated the syntax rather than adding complex character escaping logic.  Good call on opening a ticket -- I think it's worth a discussion that includes Leyla for how much we're willing to spend on backwards API compatibility (e.g. do we need to maintain full test suites on all API versions, etc...)

Reply:
FWIW I might have switched the semi-colon and pipe operators, so that it would be `facet_id:filter_value;facet_id_2:filter_value_2|filter_value_3`
I think that's because the pipe feels more like an 'or' character to me, and semi-colon is more like an 'and'.  But that's just me, and probably not worth changing at this point